index.html

```diff
@@ -0,0 +1,196 @@
+                        <span class="features__container-span">
+                            <i class="fa-solid fa-mobile-screen-button"></i
+                        ></span>
+                        <h4 class="features__header-list">FULLY RESPONSIVE</h4>
+                        <p class="features__item">
+                            Looks amazing on any device: smartphone, tablet,
+                            laptop and desktop.
+                        </p>
+                    </div>
+                    <div class="features__container-list">
+                        <span class="features__container-span">
+                            <i class="fa-solid fa-screwdriver-wrench"></i
+                        ></span>
+                        <h4 class="features__header-list">CUSTOMIZABLE</h4>
+                        <p class="features__item">
+                            Change the colors, pictures or any of the sections
+                            to suit your needs.
+                        </p>
+                    </div>
+                    <div class="features__container-list">
+                        <span class="features__container-span"
+                            ><i class="fa-solid fa-award"></i
+                        ></span>
+                        <h4 class="features__header-list">
+                            SLICK AND BEAUTIFUL DESIGN
+                        </h4>
+                        <p class="features__item">
+                            Trendy and fresh design, fits any website.
+                        </p>
+                    </div>
+                </div>
+            </section>
+            <div class="logo" id="logo">
+                <img src="./images/logos.png" alt="logo" />
+            </div>
+            <section class="simple" id="simple">
+                <div class="simple__picture"></div>
+                <di class="simple__text">
+                    <h2>SIMPLE AND BEAUTIFUL</h2>
+                    <p>
+                        Use Nova theme for your next web project. It is
+                        completely customizable so you can change any of the
+                        sections to fit your needs. Nova Theme is Free for any
+                        kind of use, personal and commercial. Have fun and good
+                        luck!
+                    </p>
+                </di>
+            </section>
+            <section class="container__pricing" id="pricing">
+                <div class="pricing__header">
+                    <h2>CHOOSE YOUR PRICING PLAN</h2>
+                    <p>Pick any of our super affordable pricing plans</p>
+                </div>
+                <div class="pricing__header-items">
+                    <div class="pricing__item-first">
+                        <h3>BASIC</h3>
+                        <h2 class="pricing__header-price">$15</h2>
+                        <p>per month</p>
+                        <p>Up to 7 Project 2 Additional Developers</p>
```

Comment:
Zdecydowanie lepiej zrobić to jako `ul > li` bo to lista elementów w tym pakiecie cenowym